### PR TITLE
fix(content) Added missing quotes in various conversations

### PR DESCRIPTION
**Pull request by Pshy0**
**Bugfix:** This PR adds missing quotes in various conversations.

## Fix Details
I used this script to find ~~and add~~ (many false positives) those missing quotes:
```sh
#!/bin/sh
grep -R -E '^[^"]*"(([^"]*"){2})*[^"]*$' ./data/ | sed '/`"`/d' | sed "s/:.*$//" | sort | uniq | sed -E 's/(^.*$)/"\1"/' > MISSQUOTES.tmp
cat MISSQUOTES.tmp | xargs sed -E 's/^(\t*`[^"]*"[A-Za-Z0-9](([^"]*"){2})*[^"]*)`$/\1`#MISSQUOTE/' -i
cat MISSQUOTES.tmp | xargs sed -E 's/^(\t+`(\t||  +)[^ ].*)`#MISSQUOTE$/\1"`/' -i
cat MISSQUOTES.tmp | xargs sed -E 's/#MISSQUOTE$//' -i
```

## Testing Done
- I have read the diff to check for improperly handled cases (i did not find one).
- I ran the game, no warnings, no smoke.

### CODE REVIEW

**Comment by warp-core** [comment]
Something to note: we don't include a closing quotation mark on a paragraph when the next one begins with the same speaker speaking, and it looks like many of these changes are those cases.

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -97,7 +97,7 @@ mission "Remnant: Cognizance 2"
 				`	"Thanks for the explanation!"`
 					goto acceptance
 				`	"I thought prefects were specialists."`
-			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the suggestions from other prefects and the specialists in the fields relevant to the task.`
+			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the suggestions from other prefects and the specialists in the fields relevant to the task."`
```
Comment (Zitchas) [inline data/remnant/remnant 2 cognizance.txt:100]: The enthusiasm is awesome, but as Warpcore said, there are some cases where quotation marks are intentionally not placed. This is one such instance. More precisely, this is an example of a paragraph break *within* a continuous block of speech by a single person.

**Comment by Pshy0** [comment]
> Something to note: we don't include a closing quotation mark on a paragraph when the next one begins with the same speaker speaking, and it looks like many of these changes are those cases.

Ok, didnt know that

**Comment by Amazinite** [comment]
http://baxtercommunications.nl/quotation-marks-multi-paragraph-rule/

File: data/kahet/kahet missions.txt [hunk 2]
```diff
@@ -421,14 +421,14 @@ mission "Ka'het: Patir Mystery 1"
...
+			`	"They do not have any drives on board, not even a hyperdrive. Limited to sub-light velocity, they would need hundreds of years to pass from one system to another, which is a testament to the Builders' patience. Most species we have encountered abandon any form of sub-light transportation between star systems immediately after developing hyperdrive technology. For all we know, the Ember Waste could have been visited by these Fetri'sei or some other Ka'sei, but they had to leave for some reason we are not aware of. The last Builder died almost 100,000 years ago - so long that having a clear idea of what happened is impossible."`
```
Comment (Amazinite) [inline data/kahet/kahet missions.txt:428]: This one is correct.

File: data/kahet/kahet missions.txt [hunk 3]
```diff
@@ -597,21 +597,21 @@ mission "Ka'het: Patir Mystery 5"
 	on offer
 		"reputation: Ka'sei" = -10
 		conversation
-			`As you approach the structure, Dusk stops preparing the equipment to observe it from the window. "It is a fascinating structure. I've only seen a small number of Builder buildings, but this looks different from anything I have seen. Perhaps it is more structurally similar to their stations than their outposts given the almost negligible gravity, or they had other asteroid colonies that have been gradually lost over time. It would be worth doing a serious exploration of the asteroid belts in the Graveyard at some point.`
+			`As you approach the structure, Dusk stops preparing the equipment to observe it from the window. "It is a fascinating structure. I've only seen a small number of Builder buildings, but this looks different from anything I have seen. Perhaps it is more structurally similar to their stations than their outposts given the almost negligible gravity, or they had other asteroid colonies that have been gradually lost over time. It would be worth doing a serious exploration of the asteroid belts in the Graveyard at some point."`
```
Comment (Amazinite) [inline data/kahet/kahet missions.txt:600]: This one is correct.

File: data/hai/hai reveal 5 philosophers.txt [hunk 4]
```diff
@@ -1324,7 +1324,7 @@ mission "Hai Reveal [B13] Freedom"
...
-			`	The Hai speaks. "We are the leaders of the civilian government of Elenchus. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your civil war. Elenchus is an independent nation, and we intend to stay so unless you intend to force otherwise. I can, however, promise that unlike our overlords, we have no interest in anything other than peace. We will send no military ships through the wormhole.`
+			`	The Hai speaks. "We are the leaders of the civilian government of Elenchus. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your civil war. Elenchus is an independent nation, and we intend to stay so unless you intend to force otherwise. I can, however, promise that unlike our overlords, we have no interest in anything other than peace. We will send no military ships through the wormhole."`
```
Comment (Amazinite) [inline data/hai/hai reveal 5 philosophers.txt:1327]: This one is correct.

File: data/human/human missions.txt [hunk 5]
```diff
@@ -3744,7 +3744,7 @@ mission "Mafia Extortion"
 			`A man in a tailored suit, sunglasses, shiny shoes, and green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`
 			`	You notice that he is missing a finger on each hand.`
 			choice
-				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of speaking?`
+				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of speaking?"`
```
Comment (Amazinite) [inline data/human/human missions.txt:3747]: This one is correct.

File: data/human/human missions.txt [hunk 6]
```diff
@@ -4582,7 +4582,7 @@ mission "Saving Artifacts 2"
 			choice
 				`	"Never mind, I'll pay."`
 					goto pay
-				`	"All right, I'll fight him.`
+				`	"All right, I'll fight him."`
```
Comment (Amazinite) [inline data/human/human missions.txt:4585]: Correct.

File: data/human/human missions.txt [hunk 7]
```diff
@@ -4783,7 +4783,7 @@ mission "Dangerous Games 2"
 			`	A crude sign behind him reads, 'Dangerous Games - Karengo's Galactic Adventures'.`
 			`	"Like it?" Karengo follows your gaze. "Decided to make this an official thing. Well... an unofficial official thing. Feds wouldn't understand. Doesn't matter; I need a pilot again, and a good one. Your ship's waterproof, right?"`
 			choice
-				`	"Uh, yeah. I think so.`
+				`	"Uh, yeah. I think so."`
```
Comment (Amazinite) [inline data/human/human missions.txt:4786]: Correct.

File: data/human/human missions.txt [hunk 8]
```diff
@@ -5793,7 +5793,7 @@ mission "Argosy Hijacking"
 				`	(Find the captain.)`
 				`	(Don't worry about it.)`
 					decline
-			`	You find him in the spaceport lounge. As you walk in, he recognizes you and calls you over. "How are you doing? Did you recover the <npc>?`
+			`	You find him in the spaceport lounge. As you walk in, he recognizes you and calls you over. "How are you doing? Did you recover the <npc>?"`
```
Comment (Amazinite) [inline data/human/human missions.txt:5796]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 9]
```diff
@@ -836,34 +836,34 @@ mission "Remnant: Technology Available"
 			label onwards
 			`	As he talks, he gestures for you to walk with him towards a nearby hall. Inside, you discover that it is one of the many cafeterias that seem to be common in Remnant starports. He picks up two trays from the window and leads you to an empty table. "While you are welcome to become one of us, we suspect that you are a citizen of the galaxy - wandering hither and yon building coalitions and enmities with species and groups across the breadth of accessible space, unfettered by conventional allegiances. As such, we hope that you will continue to explore and stop by to let us know what you have found. We ask only that you keep the secret with which you are trusted and bring us word of anything that might influence humanity's long-term survival."`
 			choice
-				`	"What secret do you want me to keep?`
+				`	"What secret do you want me to keep?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:839]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 10]
```diff
@@ -836,34 +836,34 @@ mission "Remnant: Technology Available"
...
+			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity, or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite."`
 				goto feast
 			label honored
 			action
 				set "remnant: desired to join them"
 			`	He nods, smiling. "You are not there yet, but I think you might well become one of us before too long." He takes a bite from his plate, then continues. "Keep learning, and you may well earn yourself a place among us."`
 			choice
-				`	"What secret do you want me to keep?`
+				`	"What secret do you want me to keep?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:854]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 11]
```diff
@@ -836,34 +836,34 @@ mission "Remnant: Technology Available"
...
 					goto secret
 				`	"What constitutes 'things that influence humanity's survival'?"`
 					goto survival
 			label traveler
 			`	He nods. "Thank you for confirming our impression. We are grateful to have you with us, and look forward to hearing about your travels."`
 			choice
-				`	"What secret do you want me to keep?`
+				`	"What secret do you want me to keep?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:861]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 12]
```diff
@@ -836,34 +836,34 @@ mission "Remnant: Technology Available"
...
 				`	"While I have enjoyed my time among you, I can never stay in one place."`
 					goto traveler
 				`	"What constitutes 'things that influence humanity's survival'?"`
 			label survival
-			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity, or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite.`
+			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity, or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite."`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:847]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 13]
```diff
@@ -1538,7 +1538,7 @@ mission "Remnant: Deep Surveillance"
...
 					goto leave
 			
-			`	He turns his chair back to you and stands up. "News flash, pal. I'm on to you. You asked for the research I had back before we had entered the wormhole in Terminus, and then you later told me that you'd already been through there and met these Remnant. Then when I asked you to help me learn more about them, you said no. I think you're covering for them in some way and giving them information on what I know. Why you're doing that, I don't know, but that's what my gut says, and I've learned to trust my gut on feelings like these. Now get out of my office.`
+			`	He turns his chair back to you and stands up. "News flash, pal. I'm on to you. You asked for the research I had back before we had entered the wormhole in Terminus, and then you later told me that you'd already been through there and met these Remnant. Then when I asked you to help me learn more about them, you said no. I think you're covering for them in some way and giving them information on what I know. Why you're doing that, I don't know, but that's what my gut says, and I've learned to trust my gut on feelings like these. Now get out of my office."`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:1541]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 14]
```diff
@@ -1928,7 +1928,7 @@ mission "Remnant: D94-YV Shield Generator"
 			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely applying some kind of material over a hole in a hull. "Greetings, <first>. What brings you here today?" She reseals the hatch, and drops to the ground on a tether. "Did you bring us something?"`
 				goto next
 			label familiar
-			`	You decide that you are more likely to find her in the shipyard, so you skip the starport and head out among the ships. As you search through the yard you are soon approached by another engineer. "Well met, Captain. Taely said you may be bringing back samples of new technology from ancestral space. Do you have something to show us?`
+			`	You decide that you are more likely to find her in the shipyard, so you skip the starport and head out among the ships. As you search through the yard you are soon approached by another engineer. "Well met, Captain. Taely said you may be bringing back samples of new technology from ancestral space. Do you have something to show us?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:1931]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 15]
```diff
@@ -3009,12 +3009,12 @@ mission "Remnant: Face to Maw 2B"
...
 			label main
-			`	He leads you to a room that looks like an armory where a team of technicians is putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we have not had the key resources for making Void Rifles in centuries. Fortunately, we have not really needed them since we resolved the threats that originally plagued our settlements.`
+			`	He leads you to a room that looks like an armory where a team of technicians is putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we have not had the key resources for making Void Rifles in centuries. Fortunately, we have not really needed them since we resolved the threats that originally plagued our settlements."`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3012]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 16]
```diff
@@ -3650,7 +3650,7 @@ mission "Remnant: Expanded Horizons Astral 2"
 			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, tests, and observations. After a few hours of work, she reappears on the bridge.`
 			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally, and we are creating extensive archives of astronomical data as well. Another few hours, and we should have all the data we need."`
 			choice
-				`	"Good work. What kind of data are you recording?`
+				`	"Good work. What kind of data are you recording?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3653]: Correct.

File: data/remnant/remnant 1 introduction.txt [hunk 17]
```diff
@@ -3874,7 +3874,7 @@ mission "Remnant: Logistics 1"
...
+			`	He smiles and responds, "Very good! Very good! We schedule everything, but we maintain slack in the system to take advantage of opportunities like the power system being ready early, or to deal with emergencies. Sometimes, of course, we have to drop everything and sort the schedule out later. But, in normal times, we publish these jobs on the job board and captains that have capacity available can pick them up. The projects provide resource allocations to cover operating and crew costs, but there is no... what is the word? Margin, yes! Since your ship is not in our planned resources, you provide extra capacity, and we might be able to move some of our larger projects forward when you are in our space."`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3877]: Correct.

File: data/human/deep missions.txt [hunk 18]
```diff
@@ -3387,7 +3387,7 @@ mission "Mani Refit"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from the Transport Union:`
-			`	"The fleet is ready for its inspirational journey. Please return to Mani.`
+			`	"The fleet is ready for its inspirational journey. Please return to Mani."`
 			`		-Jayden, Mani Administrator, Transport Union"`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3391]: Remove the quotation marks from both of these lines. The opening quotation mark on this first line is closed by the one on the second line, but this is a text message, not a spoken quote, so it shouldn't be quoted.

File: data/human/deep missions.txt [hunk 19]
```diff
@@ -3734,7 +3734,7 @@ mission "Home for Skadenga 10"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from Hroar on <planet>.`
-			`	"Everything is ready. Get back here.`
+			`	"Everything is ready. Get back here."`
 			`				-Hroar`
 			`			CEO of Hardy Hroar's Haulers`
 			`			'We're an unstoppable force that has never met an immovable object!'"`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3740]: Same story here, as well as the single quotation marks.

File: data/human/deep missions.txt [hunk 20]
```diff
@@ -4058,7 +4058,7 @@ mission "Hjlod Remembers Windblain"
 	on offer
 		dialog
 			`Your inbox hums with a new message from <planet>:`
-			`	"Dear <first>,`
+			`	"Dear <first>,"`
 			`	I never properly thanked you for saving my life on Nifel all those years ago. If you're ever in the neighborhood, please stop by <planet>.`
 			`				Signed, Hjlod"`
```
Comment (Amazinite) [inline data/human/deep missions.txt:4063]: Ditto.

File: data/human/deep missions.txt [hunk 21]
```diff
@@ -4088,7 +4088,7 @@ mission "Hjlod Remembers Nifel"
 	on offer
 		dialog
 			`Your inbox hums with a new message from Nifel.`
-			`	"Dear <first>,`
+			`	"Dear <first>,"`
 			`	I never properly thanked you for saving my life all those years ago. If you're ever in the neighborhood, please stop by <planet>.`
 			`				Signed, Hjlod"`
```
Comment (Amazinite) [inline data/human/deep missions.txt:4093]: Ditto.

**Review by Amazinite — CHANGES_REQUESTED** [review]
Commented on the correct ones. The rest need to go.

File: data/human/deep missions.txt (hunk 18 above)
Comment (Pshy0) [reply data/human/deep missions.txt]: In this case, how do you identify what is being quoted as part of that text message? Indentation? If so, shouldn't it be indented further, for cases (or only in those cases?), where the line above the quote is also indented, due to it not being the first line of the conversation?

Comment (Amazinite) [reply data/human/deep missions.txt]: This is the style of more "formal" written messages that the player is reading. 
https://github.com/endless-sky/endless-sky/blob/9aaa25867640a24f2d5450698ba682feacb1526f/data/human/free%20worlds%200%20prologue.txt#L2070-L2079

Comment (Pshy0) [reply data/human/deep missions.txt]: Alright, I am removing the quotes for those cases.
About indentation, in this case, there is only two levels before the signatures, should this be fixed in this PR?

File: data/human/deep missions.txt (hunk 20 above)
Comment (Pshy0) [reply data/human/deep missions.txt]: Shouldn't `"Dear <first>,"` loose it's indentation?

File: data/human/deep missions.txt (hunk 21 above)
Comment (Pshy0) [reply data/human/deep missions.txt]: Shouldn't `"Dear <first>,"` loose it's indentation?

File: data/human/deep missions.txt [hunk 22]
```diff
@@ -4088,9 +4088,9 @@ mission "Hjlod Remembers Nifel"
 	on offer
 		dialog
 			`Your inbox hums with a new message from Nifel.`
-			`	"Dear <first>,`
+			`	Dear <first>,`
```
Comment (Amazinite) [inline data/human/deep missions.txt:4091]:
```suggestion
			``
			`Dear <first>,`
```

File: data/human/deep missions.txt [hunk 23]
```diff
@@ -4058,9 +4058,9 @@ mission "Hjlod Remembers Windblain"
 	on offer
 		dialog
 			`Your inbox hums with a new message from <planet>:`
-			`	"Dear <first>,`
+			`	Dear <first>,`
```
Comment (Amazinite) [inline data/human/deep missions.txt:4061]:
```suggestion
			``
			`Dear <first>,`
```

File: data/human/deep missions.txt [hunk 24]
```diff
@@ -3734,10 +3734,10 @@ mission "Home for Skadenga 10"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from Hroar on <planet>.`
-			`	"Everything is ready. Get back here.`
+			`	Everything is ready. Get back here.`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3737]:
```suggestion
			``
			`	Everything is ready. Get back here.`
```

File: data/human/deep missions.txt [hunk 25]
```diff
@@ -3387,8 +3387,8 @@ mission "Mani Refit"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from the Transport Union:`
-			`	"The fleet is ready for its inspirational journey. Please return to Mani.`
-			`		-Jayden, Mani Administrator, Transport Union"`
+			`	The fleet is ready for its inspirational journey. Please return to Mani.`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3390]:
```suggestion
			``
			`	The fleet is ready for its inspirational journey. Please return to Mani.`
```

File: data/human/deep missions.txt [hunk 26]
```diff
@@ -3387,8 +3387,8 @@ mission "Mani Refit"
...
+			`		-Jayden, Mani Administrator, Transport Union`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3391]:
```suggestion
			`				Jayden, Mani Administrator, Transport Union`
```

File: data/human/deep missions.txt [hunk 27]
```diff
@@ -3734,10 +3734,10 @@ mission "Home for Skadenga 10"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from Hroar on <planet>.`
-			`	"Everything is ready. Get back here.`
+			`	Everything is ready. Get back here.`
 			`				-Hroar`
```
Comment (Amazinite) [inline data/human/deep missions.txt:3738]:
```suggestion
			`				Hroar`
```

**Review by Amazinite — APPROVED** [review]

**Pull request merged**